packages/react-data-grid/src/Canvas.tsx

```diff
@@ -110,6 +110,12 @@ export default function Canvas<R, K extends keyof R>({
     }
   }, [rowHeight, scrollToRowIndex]);
 
+  useEffect(() => {
+    if (scrollTop + clientHeight - rowHeight * rowsCount === 8) {
```

Comment:
`8` is corresponding to this line.

https://github.com/adazzle/react-data-grid/pull/1824/files#diff-ee619d375026eab3db5bbae675917325R32